### PR TITLE
trilium notes

### PR DESCRIPTION
**Pull request by zadam**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.github.zadam.trilium.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+      - rm -r $SRC_DIR/src/public/app-dist/*.mobile.*
+      - cp -r bin/better-sqlite3/linux-desktop-better_sqlite3.node $SRC_DIR/node_modules/better-sqlite3/build/Release/better_sqlite3.node
+
+      - ./node_modules/.bin/electron-packager $SRC_DIR --asar --out=dist --executable-name=trilium --platform=linux --arch=x64 --overwrite
+
+      - rm -r "dist/Trilium Notes-linux-x64/swiftshader"
+      - cp bin/tpl/anonymize-database.sql "dist/Trilium Notes-linux-x64/"
+
+      - cp -r "dist/Trilium Notes-linux-x64" /app/trilium
```
Comment (hfiguiere) [inline com.github.zadam.trilium.yml:92]: so what do you do for the aarch64 build?

File: com.github.zadam.trilium.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+
+command: trilium
+
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --socket=wayland
```
Comment (hfiguiere) [inline com.github.zadam.trilium.yml:18]: electron really work with Wayland?

File: com.github.zadam.trilium.yml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  - --socket=fallback-x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
```
Comment (hfiguiere) [inline com.github.zadam.trilium.yml:23]: does it really need access to the whole home directory?

**Review by hfiguiere — CHANGES_REQUESTED** [review]
Have the .desktop and .metainfo files be submitted upstream?

**Comment by hfiguiere** [comment]
bot, build com.github.zadam.trilium

File: com.github.zadam.trilium.yml [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+      - (cd $SRC_DIR && npm install --only=prod --offline)
+
+      - sed -i -e 's/app\/desktop.js/app-dist\/desktop.js/g' $SRC_DIR/src/views/desktop.ejs
+      - sed -i -e 's/app\/mobile.js/app-dist\/mobile.js/g' $SRC_DIR/src/views/mobile.ejs
+      - sed -i -e 's/app\/setup.js/app-dist\/setup.js/g' $SRC_DIR/src/views/setup.ejs
+
+      - rm -r $SRC_DIR/src/public/app-dist/*.mobile.*
+      - cp -r bin/better-sqlite3/linux-desktop-better_sqlite3.node $SRC_DIR/node_modules/better-sqlite3/build/Release/better_sqlite3.node
+
+      - ./node_modules/.bin/electron-packager $SRC_DIR --asar --out=dist --executable-name=trilium --platform=linux --arch=x64 --overwrite
```
Comment (hfiguiere) [inline com.github.zadam.trilium.yml:87]: `x64` that x86_64. That's where the aarch64 build is hanging.

File: com.github.zadam.trilium.yml (hunk 1 above)
Comment (zadam) [reply com.github.zadam.trilium.yml:92]: We're not actually providing aarch64 desktop builds, is that mandatory?

File: com.github.zadam.trilium.yml (hunk 3 above)
Comment (zadam) [reply com.github.zadam.trilium.yml:23]: The app has user-configurable directory where it stores its data. By default it looks for data in `~/.local/share/trilium-data` and `~/trilium-data` but again, allows to store them elsewhere (configurable via env. variable). So I'm not sure how to approach this.

File: com.github.zadam.trilium.yml (hunk 2 above)
Comment (zadam) [reply com.github.zadam.trilium.yml:18]: Electron v12 and higher should work fine with wayland: https://news.ycombinator.com/item?id=26317942

**Comment by zadam** [comment]
@hfiguiere regarding .desktop and metainfo, no, they are not used in upstream. Non-flatpak app actually generates the .desktop icon in runtime. Is it a problem to have them separate / not in the upstream? I can also remove them if that would make things better.

For some context, this manifest has been created by somebody else, and I'm sort of submitting it on their behalf as the author of the app.

File: com.github.zadam.trilium.yml (hunk 1 above)
Comment (barthalion) [reply com.github.zadam.trilium.yml:92]: It's not, but probably worth to reconsider it at some point. The build can be limited to x86_64: https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64=

**Comment by barthalion** [comment]
As you are the upstream developer, we can provide some guidance (like "these files should be maintained in the main repo"), but generally speaking, it's not considered a blocker. If you find it more convenient to maintain it here, it's fine.

File: com.github.zadam.trilium.yml (hunk 1 above)
Comment (zadam) [reply com.github.zadam.trilium.yml:92]: Thanks for the hint, I've added the `only-arches` into `flathub.json`.

**Comment by zadam** [comment]
@barthalion ok, then I would propose to keep it this way for now, I assume it will be possible to change later as needed.

**Comment by barthalion** [comment]
bot, build com.github.zadam.trilium

**Comment by barthalion** [comment]
Does not launch for me:

```
~ ❱ flatpak run com.github.zadam.trilium
DB not initialized, please visit setup page
DB size: 4 KB
/app/bin/trilium: line 2:     3 Trace/breakpoint trap   (core dumped) zypak-wrapper.sh /app/trilium/trilium "$@"
~ 1.6s [133] coredumpctl info --no-pager 
           PID: 44149 (trilium)
           UID: 1000 (bpiotrowski)
           GID: 1000 (bpiotrowski)
        Signal: 5 (TRAP)
     Timestamp: Fri 2022-07-29 10:19:18 CEST (9s ago)
  Command Line: $'/app/trilium/trilium --enable-crashpad'
    Executable: /app/trilium/trilium
 Control Group: /user.slice/user-1000.slice/user@1000.service/app.slice/app-flatpak-com.github.zadam.trilium-44134.scope
          Unit: user@1000.service
     User Unit: app-flatpak-com.github.zadam.trilium-44134.scope
         Slice: user-1000.slice
     Owner UID: 1000 (bpiotrowski)
       Boot ID: ab112c596db140f4bf4ceb3ae4131807
    Machine ID: 0dcea0c5dcc74463bebcfe97d0835b1e
      Hostname: gnomuru
       Storage: /var/lib/systemd/coredump/core.trilium.1000.ab112c596db140f4bf4ceb3ae4131807.44149.1659082758000000.zst (present)
     Disk Size: 13.8M
       Message: Process 44149 (trilium) of user 1000 dumped core.
                
                Module /app/trilium/trilium with build-id 6df149d8ff14d9d2f9510ea1ae8c0df9263c01b6
                Module /tmp/.org.chromium.Chromium.2viLC7 with build-id c803b7fec7b063af3bf4ddef3f06d3647bffc3b5
                Module /usr/lib/x86_64-linux-gnu/libbrotlicommon.so.1.0.9 with build-id eb87466e7fa34a90e0755f21e023d3db0a987ce2
                Module /usr/lib/x86_64-linux-gnu/libgpg-error.so.0.33.0 with build-id c30db66ea0686f6ee2da427bb72a2e89e6090583
                Module /usr/lib/x86_64-linux-gnu/libdatrie.so.1.4.0 with build-id 7803f658826f99ddcdc078f28b18b422c0bfec58
                Module /usr/lib/x86_64-linux-gnu/libbrotlidec.so.1.0.9 with build-id a98d9d21ed529d40fc9b6270664e4bc96e29526b
                Module /usr/lib/x86_64-linux-gnu/libbz2.so.1.0.8 with build-id 384a34538f283d6f35efa370bb781db34ecd0989
                Module /usr/lib/x86_64-linux-gnu/libgraphite2.so.3.2.1 with build-id 6f2e6e8dffff370a964ee4d63441ba97297954b7
                Module /usr/lib/x86_64-linux-gnu/libgmp.so.10.4.0 with build-id 74fbee638a80df912f01d75b2be3398d7186243c
                Module /usr/lib/x86_64-linux-gnu/libhogweed.so.6.4 with build-id da2e1a3c32a622eaadc09325ae6ddeddbdab16da
                Module /usr/lib/x86_64-linux-gnu/libnettle.so.8.4 with build-id c5330a4ecc25167802fa655726f88a5ca5bb2619
                Module /usr/lib/x86_64-linux-gnu/libtasn1.so.6.6.2 with build-id 2cd130a9b1864418413e41daaf17745b858cabc9
                Module /usr/lib/x86_64-linux-gnu/libunistring.so.2.1.0 with build-id b081092317b45d39c72a209bac13f85375b19da3
                Module /usr/lib/x86_64-linux-gnu/libidn2.so.0.3.7 with build-id a65a0e54327f3f66c006d950adb03596844c1724
                Module /usr/lib/x86_64-linux-gnu/libp11-kit.so.0.3.0 with build-id f2fd7d4d26aedc34cb5063ba89349ec45ec6e137
                Module /usr/lib/x86_64-linux-gnu/libblkid.so.1.1.0 with build-id d2e2ab2d5d15769243b7e2c6a520681dfd108f85
                Module /usr/lib/x86_64-linux-gnu/libgcrypt.so.20.2.10 with build-id 0d6d2a15b73063b6778803c452930751b8aabab7
                Module /usr/lib/x86_64-linux-gnu/libcap.so.2.59 with build-id ab63c8e822cccba1aa915a4861cec37c765f16d6
                Module /usr/lib/x86_64-linux-gnu/liblz4.so.1.9.3 with build-id 68089aa38d4540fbb03ed3ce0532b62781de35c9
                Module /usr/lib/x86_64-linux-gnu/liblzma.so.5.2.5 with build-id ae62e4270d4dc069879b237a837ef2859efd09eb
                Module /usr/lib/x86_64-linux-gnu/libXau.so.6.0.0 with build-id 79d75a43cb3ee378c22b04a786c0180327264f58
                Module /usr/lib/x86_64-linux-gnu/libwayland-server.so.0.20.0 with build-id 6a102ad776418ac13f48100d221904e6657b079b
                Module /usr/lib/x86_64-linux-gnu/libjpeg.so.62.3.0 with build-id efa8a5e8386843c097215ffc0c8ec0ed4fa01c90
                Module /usr/lib/x86_64-linux-gnu/libXrender.so.1.3.0 with build-id a9208af275a7b2dfbf1c0051529f923733fe6193
                Module /usr/lib/x86_64-linux-gnu/libxcb-render.so.0.0.0 with build-id 9b81b542b9efa68697a1d904c7bd257c6f2b3b83
                Module /usr/lib/x86_64-linux-gnu/libxcb-shm.so.0.0.0 with build-id 0fa704f4bfb9d660e08133465656c673510102a7
                Module /usr/lib/x86_64-linux-gnu/libpng16.so.16.37.0 with build-id 1dad6c71acfbe987e3b0104862c1a3546c8e7ebe
                Module /usr/lib/x86_64-linux-gnu/libpixman-1.so.0.40.0 with build-id a0b6c539dba70b320e743093df79a978655220a3
                Module /usr/lib/x86_64-linux-gnu/libthai.so.0.3.1 with build-id 1478bcc58e99dcd53bb147925903e96e333afe79
                Module /usr/lib/x86_64-linux-gnu/libfreetype.so.6.18.0 with build-id 0c8c366db56512257f24ae93d849c392233860f2
                Module /usr/lib/x86_64-linux-gnu/libfontconfig.so.1.12.0 with build-id 8c9de74ee5e58a9633b18b0efd6a8a7f266e91b3
                Module /usr/lib/x86_64-linux-gnu/libharfbuzz.so.0.30000.0 with build-id ccebcd32ec1b116ee2ddfece93ebe7ab80fd1b79
                Module /usr/lib/x86_64-linux-gnu/libpangoft2-1.0.so.0.4800.11 with build-id 47b0204cfcdd1df465aa1f6dadfa9faedc1d45c2
                Module /usr/lib/x86_64-linux-gnu/libfribidi.so.0.4.0 with build-id 0119982583d1da53f547d866260938c99a0f0deb
                Module /usr/lib/x86_64-linux-gnu/libepoxy.so.0.0.0 with build-id 87e8d9a0103e3674ba958dfa717333757cb9ecca
                Module /usr/lib/x86_64-linux-gnu/libwayland-client.so.0.20.0 with build-id 682bdfbd49712510f363ee5878042646d0c4c123
                Module /usr/lib/x86_64-linux-gnu/libwayland-egl.so.1.20.0 with build-id 3fb56c7b561bce42d553d52f49bd61bd18c24743
                Module /usr/lib/x86_64-linux-gnu/libwayland-cursor.so.0.20.0 with build-id 97398aa2754c08ba4f47e1c5c9182425eba4482b
                Module /usr/lib/x86_64-linux-gnu/libcairo-gobject.so.2.11600.0 with build-id a65133924b04f19635ba778cc66bc1e75e337ffd
                Module /usr/lib/x86_64-linux-gnu/libXi.so.6.1.0 with build-id 9f055e366d658955104ff7437eb4d48a4fafdd19
                Module /usr/lib/x86_64-linux-gnu/libpangocairo-1.0.so.0.4800.11 with build-id 878214c04a0023228caa75067b849434839b9d07
                Module /usr/lib/x86_64-linux-gnu/librt-2.33.so with build-id b6408705e6ec4347ab6399c880847a98af3b1c0e
                Module /usr/lib/x86_64-linux-gnu/libXinerama.so.1.0.0 with build-id 7149a14348b88eafe4df033af8b0562a2a07412e
                Module /usr/lib/x86_64-linux-gnu/libgdk-3.so.0.2404.30 with build-id fe69ece3a00a07fd18559e7783502366f908bacb
                Module /usr/lib/x86_64-linux-gnu/libgnutls.so.30.30.0 with build-id e46dfac77f54d1d788dded3a3aadc2a130e2e771
                Module /usr/lib/x86_64-linux-gnu/libplc4.so with build-id 475285a5c7d757b7f8ed1e8835ddd4deee90e764
                Module /usr/lib/x86_64-linux-gnu/libplds4.so with build-id 358aeb203dd1e86567c6e6c0c90ab7879d605bd5
                Module /usr/lib/x86_64-linux-gnu/libresolv-2.33.so with build-id 37fbb9160d1a1f7d122eba2548b81e405dc3575e
                Module /usr/lib/x86_64-linux-gnu/libmount.so.1.1.0 with build-id b6f52f2cc03315e310379ad8526e8dce95dc7497
                Module /usr/lib/x86_64-linux-gnu/libz.so.1.2.11 with build-id 5885119a2a8097b900bdb4ca40560e892ae7ac8c
                Module /usr/lib/x86_64-linux-gnu/libpcre.so.1.2.13 with build-id 24b84a4458ccb5a5ea71c745c388f11e6a35fb20
                Module /usr/lib/x86_64-linux-gnu/libffi.so.7.1.0 with build-id 9409f4a8a0ef373035bf02ad335a73a0970e16b7
                Module /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.29 with build-id 3ccb527c74d16a861ad336ace3987a72b5ffc20b
                Module /usr/lib/x86_64-linux-gnu/libsystemd.so.0.32.0 with build-id 733d173085982a2267105faeaaa0b3329e243ab3
                Module /usr/lib/x86_64-linux-gnu/libc-2.33.so with build-id 833fec3acaa8ba63dfea79852744153ba716f8cb
                Module /usr/lib/x86_64-linux-gnu/libgcc_s.so.1 with build-id 6dca6d6f8e253228cab3d4469a04548560f18ce4
                Module /usr/lib/x86_64-linux-gnu/libatspi.so.0.0.1 with build-id 39093783b92e863298ab5e052ee1157d0fbe7c8d
                Module /usr/lib/x86_64-linux-gnu/libasound.so.2.0.0 with build-id cea439ce6bbabab50dfb933129948c8652c885ae
                Module /usr/lib/x86_64-linux-gnu/libxkbcommon.so.0.0.0 with build-id b1e4a501eeeaeb9df2aad95ab9a3c0e5a98b58dc
                Module /usr/lib/x86_64-linux-gnu/libxcb.so.1.1.0 with build-id b5353f38eb1aa2da400913a09ab73f491d3112eb
                Module /usr/lib/x86_64-linux-gnu/libexpat.so.1.8.6 with build-id 5384de1016ef1d625d6dd13fcbb73fa697853f4d
                Module /usr/lib/x86_64-linux-gnu/GL/default/lib/libgbm.so.1.0.0 with build-id 973fd512ece4eea7da265f332d9d85c2d5c4c3b3
                Module /usr/lib/x86_64-linux-gnu/libXrandr.so.2.2.0 with build-id d06b7bfdd22a4c1ab613440a0e098de621804ea0
                Module /usr/lib/x86_64-linux-gnu/libXfixes.so.3.1.0 with build-id 8a9a9cdb2f8f462546a388cd220d6b491e73b3a0
                Module /usr/lib/x86_64-linux-gnu/libXext.so.6.4.0 with build-id f31779e34fecba641fa6cc43d70c20283b6177b8
                Module /usr/lib/x86_64-linux-gnu/libXdamage.so.1.1.0 with build-id a1afed4afa3204726d7c48a960b7e7b8b0158dd2
                Module /usr/lib/x86_64-linux-gnu/libXcomposite.so.1.0.0 with build-id ccafe8933554c191405b4600366d680cb8d287a8
                Module /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0 with build-id 26ed6b841c4c914418a6000492aec3d2624f7b3a
                Module /usr/lib/x86_64-linux-gnu/libm-2.33.so with build-id 3228889cbf4491261dd899e700cc36c48b65df2d
                Module /usr/lib/x86_64-linux-gnu/libgdk_pixbuf-2.0.so.0.4200.8 with build-id 04d80c9bdc96eb079d3f3b4a3c2e6d13d6f6ad55
                Module /usr/lib/x86_64-linux-gnu/libcairo.so.2.11600.0 with build-id 5ecfcc4ad9dea6892d16306dd3c5f4615ac3defc
                Module /usr/lib/x86_64-linux-gnu/libpango-1.0.so.0.4800.11 with build-id b8960bec44b1afd340d1eabba2e481fe3ab6bc89
                Module /usr/lib/x86_64-linux-gnu/libgtk-3.so.0.2404.30 with build-id 9e25293f13e6159aad26e9b6a245b7220f24f940
                Module /usr/lib/x86_64-linux-gnu/libdrm.so.2.4.0 with build-id 30e7b45b7ad83e38eee2ae8522e82c3b1f545673
                Module /usr/lib/x86_64-linux-gnu/libdbus-1.so.3.19.14 with build-id 1fdba8f3bf9a0625b76c1e012f1c2b2f1d3d8fae
                Module /usr/lib/x86_64-linux-gnu/libcups.so.2 with build-id 023035896cca37b44d95ead02d1ccf267ccb12ed
                Module /usr/lib/x86_64-linux-gnu/libatk-bridge-2.0.so.0.0.0 with build-id c8347a7bd5b08cff5176077fe7c5a7840208c688
                Module /usr/lib/x86_64-linux-gnu/libatk-1.0.so.0.23609.1 with build-id 73e4cd0b2ee3276faaf133076f8e0d1a12e44493
                Module /usr/lib/x86_64-linux-gnu/libnspr4.so with build-id c40f48e3cd8e0780a2a7f9e77c3f561dbf084820
                Module /usr/lib/x86_64-linux-gnu/libsmime3.so with build-id 04e4644577ecc2c84c6fc80f18b9ac139420aa1e
                Module /usr/lib/x86_64-linux-gnu/libnssutil3.so with build-id c3f2641fc4a16b8d0074f2c253078f2600316041
                Module /usr/lib/x86_64-linux-gnu/libnss3.so with build-id 51cf678815281800d5df29e59c0870492af13dd2
                Module /usr/lib/x86_64-linux-gnu/libgio-2.0.so.0.6800.4 with build-id 87b3c5617c3fa3ab2e9e42ab510537bd0b483b1b
                Module /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0.6800.4 with build-id 8c8106873e1e069487af456b7a40be2685116edd
                Module /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.6800.4 with build-id 0358983d1c619f8c8494f1506652b62983a5aa60
                Module /usr/lib/x86_64-linux-gnu/libgmodule-2.0.so.0.6800.4 with build-id 5f1a6dcad5ac0484dbc7b3f0222ae47ac9408f38
                Module /usr/lib/x86_64-linux-gnu/libpthread-2.33.so with build-id 97172665d1042add3adc9b120b0760258ba0aba5
                Module /usr/lib/x86_64-linux-gnu/libdl-2.33.so with build-id 20be4ecfd5346af55ccafadf4211c2c47c496b2b
                Module /app/trilium/libffmpeg.so with build-id ab03ff6062f91c8c9449708b038c1f858049678d
                Module /app/lib/libzypak-preload-host-spawn-strategy.so with build-id 1dc2aafa5d34b1dab762934fd76dd881e5dc6245
                Module /app/lib/libzypak-preload-host.so with build-id 0e53719b8a0538bb9df92294c92bfb28e884d41e
                Module /usr/lib/x86_64-linux-gnu/ld-2.33.so with build-id 856401969318527d829336ec391155c2ebc785c5
                Module linux-vdso.so.1 with build-id e319e5561ecedad979e51ac55b99f328540f2c82
                Stack trace of thread 3:
                #0  0x00005597f98f0c36 n/a (/app/trilium/trilium + 0x29bac36)
                #1  0x00005597f988eea7 n/a (/app/trilium/trilium + 0x2958ea7)
                #2  0x00005597fcc7b846 n/a (/app/trilium/trilium + 0x5d45846)
                #3  0x00005597fcc7b7e1 n/a (/app/trilium/trilium + 0x5d457e1)
                #4  0x00005597fae264c1 n/a (/app/trilium/trilium + 0x3ef04c1)
                #5  0x00005597fae26f3c n/a (/app/trilium/trilium + 0x3ef0f3c)
                #6  0x00005597fae22c89 n/a (/app/trilium/trilium + 0x3eecc89)
                #7  0x00005597f9ed9364 n/a (/app/trilium/trilium + 0x2fa3364)
                #8  0x00005597f9ed8d8f n/a (/app/trilium/trilium + 0x2fa2d8f)
                #9  0x00005597f9ed66cf n/a (/app/trilium/trilium + 0x2fa06cf)
                #10 0x00005597f9ed70b9 n/a (/app/trilium/trilium + 0x2fa10b9)
                #11 0x00005597f8f0c02c n/a (/app/trilium/trilium + 0x1fd602c)
                #12 0x00007fada6ee1bbd n/a (/app/lib/libzypak-preload-host-spawn-strategy.so + 0xb5bbd)
                #13 0x00007fada51a9b80 n/a (/usr/lib/x86_64-linux-gnu/libc-2.33.so + 0x23b80)
                #14 0x00007fada6ee1e46 n/a (/app/lib/libzypak-preload-host-spawn-strategy.so + 0xb5e46)
                #15 0x00005597f8ba166a n/a (/app/trilium/trilium + 0x1c6b66a)
                ELF object binary architecture: AMD x86-64
```

**Comment by zadam** [comment]
Hi, @barthalion, it did work for me, after some googling it appears to be connected with [wayland support](https://github.com/electron/electron/issues/32436). 

After disabling wayland socket, it appears to work under wayland (presumably under xwayland). Please retest, thanks!

**Comment by hfiguiere** [comment]
It's not like I didn't ask originally https://github.com/flathub/flathub/pull/3353#discussion_r929356545

**Comment by barthalion** [comment]
bot, build com.github.zadam.trilium

File: com.github.zadam.trilium.yml [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+
+command: trilium
+
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
```
Comment (barthalion) [inline com.github.zadam.trilium.yml:18]:
```suggestion
  - --socket=x11
```

Not a fallback anymore without Wayland socket.

**Pull request closed without merging**